docker-compose-rule-core/src/main/java/com/palantir/docker/compose/connection/Ports.java

```diff
@@ -48,9 +46,8 @@ public Ports(DockerPort port) {
         return ports.stream();
     }
 
-    public static Ports parseFromDockerComposePs(String psOutput, String dockerMachineIp) {
-        Preconditions.checkArgument(!Strings.isNullOrEmpty(psOutput), "No container found");
-        Matcher matcher = PORT_PATTERN.matcher(psOutput);
+    public static Ports parseFromPortInformation(String portInformation, String dockerMachineIp) {
```

Comment:
Unfortunately nearly everything in DCR is public rather than package-private - so we should do some quick github searches to find out if anyone is using this internally or externally before making such a breaking change.

Reply:
Fortunately in this case it does look safe, but worth bearing in mind!

Reply:
Ah yeah, missed that one, sorry. Can include explicit comments about usage / impact of the break in the future